Amazon AWS extension

victor-falcon opened the pull request:
## Description

Simple extension to work with Amazon AWS. Right now, it has only one command to list all your instances in EC2.

## Type of change

- New extension

## Screencast
![image](https://user-images.githubusercontent.com/238766/140780775-f372630f-a102-44da-865f-dfbbf827672f.png)

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

pernielsentikaer reviewed (changes requested):
Hi @victor-falcon 👋 

Just a few comments:

- The extension is not inside the extension folder, could you please the folder inside the extensions folder. 

- I think a README.md should be added to guide the user when installing

- After adding the right region, nothing happens and the console showing:

![LNxiSsV044GS7xjI](https://user-images.githubusercontent.com/38033107/140880357-ca0f704a-dd04-432d-a7b8-7316b98dfc45.png)


```
2021-11-09 08:26:46.756 Db Raycast[58669:2bebf2] [com.raycast.extensions.aws:index] Error Error: connect EHOSTDOWN 169.254.169.254:80 - Local (192.168.4.24:54359)
    at __node_internal_captureLargerStackTrace (node:internal/errors:464:5)
    at __node_internal_exceptionWithHostPort (node:internal/errors:642:12)
    at internalConnect (node:net:916:16)
    at defaultTriggerAsyncIdScope (node:internal/async_hooks:454:18)
    at node:net:1007:9
    at processTicksAndRejections (node:internal/process/task_queues:78:11)
```

Looking forward to trying it out 💪 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

victor-falcon commented:
Thank @pernielsentikaer for your feedback!

I just moved the extension to the proper folder, sorry for that, and updated the README file with info about how to set up your credentials. 😄

pernielsentikaer commented:
Hi @victor-falcon  👋 

I managed to set it up correctly and now it shows the right data 😄 

- The Open in Browser actions send me to the EC2 instance area, but I'm landing in "Security Groups" inside "Network & Security" - where am I supposed to land 😄 

This extension has a huge potential if done right and I personally looking forward to see it grow

victor-falcon commented:
@pernielsentikaer that's strange, I can't reproduce that 😞 , for me it's working. Any idea on how can I solve/check this?

pernielsentikaer commented:
@victor-falcon where was I supposed to land you say? Can you send me a screenshot to verify?

victor-falcon commented:
@pernielsentikaer here you have it. For my, it just goes to `https://{region}.console.aws.amazon.com/ec2/v2/home?region={region}#InstanceDetails:instanceId={instanceId}`.

![image](https://user-images.githubusercontent.com/238766/141118613-c4bdc1d0-4f5d-4d47-96be-9338e410815d.png)

pernielsentikaer commented:
Hi @victor-falcon 👋  

Found the "bug" - the toggle with "New EC2 Experience" was not enabled in chrome and therefore I landed on the other page, after checking the "New EC2 Experience" it worked as expected. I'm not sure that we can do anything about it and it should be default enabled.

I think we're ready for version  💯

PitNikola commented on extensions/amazon-aws/package.json:
```diff
@@ -0,0 +1,47 @@
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "aws",
+  "title": "Amazon AWS",
+  "description": "Integrate your AWS account in Raykcast",
```
@grzegorzkrukowski, do you mind just pushing the fix to master when you have a moment?

victor-falcon replied:
Yes please 🙏, or I can do another PR if you want

grzegorzkrukowski replied:
I will do direct push to master
